root/user/profile.tt

```diff
@@ -91,11 +91,15 @@
             [% END %]
         [% END %]
 
-        [% IF c.user_exists AND (viewing_own_profile OR c.user.is_account_admin);
-             WRAPPER property name=l('Last login:');
-               UserDate.format(user.last_login_date);
-             END;
-           END %]
+        [% IF c.user_exists AND (viewing_own_profile OR c.user.is_account_admin) %]
+           [% WRAPPER property name=l('Last login:') %]
```

Comment:
I’d still prefer it as it was before (not surprising, since I intentionally wrote it that way), but if those `[% %]` are necessary, please make them `[%~ ~%]` at least.